src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
@ckamm I think we mean the same thing, isn't what `DontAlign` would give us? The name is misleading, the documentation shows the following example:

```
someLongFunction(argument1,
    argument2);
```

The example for `AlwaysBreak` doesn't look nice I think:
```
someLongFunction(
    argument1, argument2);
```